test/helpers/sharedSteps.ts

```diff
@@ -69,44 +66,55 @@ import {
   getServiceDirPath,
   getSpellAquaPath,
   getSpellDirPath,
-  pathToTheTemplateWhereLocalEnvironmentIsSpunUp,
 } from "./paths.js";
 
-export const multiaddrs = await getMultiaddrs();
-
-export async function getMultiaddrs(): Promise<Node[]> {
-  const local =
-    fluenceEnv === "local"
-      ? addrsToNodes(
-          (
-            await fluence({
-              args: ["default", "peers", "local"],
-              cwd: pathToTheTemplateWhereLocalEnvironmentIsSpunUp,
-            })
-          )
-            .trim()
-            .split("\n"),
-        )
-      : [];
-
-  return {
-    kras: krasnodar,
-    stage: stage,
-    testnet: testNet,
-    local,
-  }[fluenceEnv];
+let localMultiaddrs: Node[] = [];
+
```

Comment:
Still don't understand the benefit from having this variable compared to previous variable. It will be updated only once, bcs update happens when `localMultiaddrs.length === 0` is met. And after the first update, this condition will be always false.

Reply:
this variable is just a cache that prevents `fluence default peers local` to be callied more then it needs to be called

previously getMultiaddrs was always called when this file is imported even if the code doesn't need this at all